packages/carbon-component-mapper/src/files/text-field.d.ts

```diff
@@ -0,0 +1,13 @@
+import { FormGroupProps } from "./form-group";
+import { UseFieldApiComponentConfig } from "@data-driven-forms/react-form-renderer";
+
+import { TextInputProps } from 'carbon-components-react';
+
+interface InternalTextFieldProps extends TextInputProps {
```

Comment:
This is not necessary you can directly reference carbon props in the `TextFieldProps` type.